docker-compose.yaml

```diff
@@ -66,7 +66,7 @@ services:
     expose:
       - 4004
     ports:
-      - '4040:4004'
+      - '4004:4004'
```

Comment:
I saw this. I thought it was mapped from TCP port 4004 to 4040 on purpose.  But I guess it was a mistake. If you are going to change this, you should change it for all:
```
$ grep 4040 *yaml
docker-compose.yaml:      - '4040:4004'
simplewallet-build-client-js.yaml:      - '4040:4004'
simplewallet-build-tp-cxx.yaml:      - '4040:4004'
simplewallet-build-tp-java.yaml:      - '4040:4004'
simplewallet-build-tp-js.yaml:      - '4040:4004'
simplewallet-build-tp-py.yaml:      - '4040:4004'
```

Reply:
Yep, that was the intention initially. I've been getting queries as to understand where did the port 4040 come from. :)  Hence, this update.

The changes for rest of the yaml files is scheduled for another PR, as they are not relevant to this PR's purpose.